docs/Guides/V2Upgrade.md

```diff
@@ -14,18 +14,14 @@ In the [SDK release](https://github.com/bcnmy/biconomy-client-sdk/releases/tag/r
 
 ### Steps to upgrade
 
-1. create a Biconomy account V1 instance and execute **`getUpdateImplementationData`** transaction.
+1. create a Biconomy account V1 instance and execute **`updateImplementationUserOp`** transaction. This includes upgrade transaction and the enabling ECDSA validation module.
```

Comment:
there are two things. If they just want to upgrade then it's two actions combined into updateImplementationUserOp

but if something else has to be batched then have to get individual data for getUpdateImplementationData and getEnableSetupModuleData and both MUST be batched along with your own transaction.